### DISCUSSION TITLE
Get the HTML Element from a Node

### DISCUSSION THREAD

**Discussion by AlessioGr** [ueberdosis/tiptap#2848] [General Questions]
In my plugin I have a Node object and I want to get the corresponding HTML Element for it.
How can I do that?

**Comment by pointnet** [comment]
You can get the DOM node if you know the prosemirror node position (getPos()) using:

 `editor.view.nodeDOM(pos: number).innerHTML`, 

IE: `editor.view.nodeDOM(0).innerHTML`
